### DISCUSSION TITLE
Remove name colors

### DISCUSSION THREAD

**Discussion by GiraffeKiller** [Chatterino/chatterino2#3641] [Ideas]
Maybe not a popular feature, but I figured I'd throw it out there. 
Would love to be able to remove all the colors from my chat and just let people's names be white. Or maybe any color?

I'm colorblind and I just prefer having everything as plain as possible. It's more of a "I'm used to it" sort of thing than a necessity. 
Thanks!

**Comment by ALazyMeme** [comment]
I feel like a colourblind mode setting would be appropriate here (no matter how much we don't need more settings).
